commons/src/destination.rs

```diff
@@ -0,0 +1,54 @@
+use core::fmt;
+use crate::runtime::AccountId;
+
+pub type BtcTransactionId = [u8; 32];
+pub type BankTransactionId = [u8; 32];
+
+/// Identifier for different destinations on VLN
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    parity_scale_codec::Decode,
+    parity_scale_codec::Encode,
+)]
+pub enum Destination {
+    // type to denote destinations on vln blockchain
+    Vln(AccountId),
+    // type to denote transactions on the bitcoin blockchain
+    Btc(BtcTransactionId),
+    // type to denote off chain bank transfer transations
+    Bank(BankTransactionId),
```

Comment:
This on the other hand will be key to support it soon, since bank information will contain personal user information that shouldn't touch the blockchain this will likely be an IPFS CID(an u64?) that refers to an encrypted document on IPFS that only the final liquidity provider on the chain can decrypt to send funds to the user's account